pkg/webhook/default_server/statefulset/validating/statefulset_create_update_handler.go

```diff
@@ -76,6 +77,13 @@ func (h *StatefulSetCreateUpdateHandler) Handle(ctx context.Context, req types.R
 		if allErrs := append(validationErrorList, updateErrorList...); len(allErrs) > 0 {
 			return admission.ErrorResponse(http.StatusUnprocessableEntity, allErrs.ToAggregate())
 		}
+
+		if obj.Spec.UpdateStrategy.RollingUpdate != nil &&
+			obj.Spec.UpdateStrategy.RollingUpdate.PodUpdatePolicy == appsv1alpha1.InPlaceOnlyPodUpdateStrategyType {
+			if err := validateTemplateInPlaceOnly(&oldObj.Spec.Template, &obj.Spec.Template); err != nil {
+				return admission.ErrorResponse(http.StatusUnprocessableEntity, fmt.Errorf("invalid template modified with InPlaceOnly: %v", err))
+			}
```

Comment:
Make it more clear:
Invalid pod template modified with InPlaceOnly update strategy. Currently, only image update is allowed for InPlaceOnly.